### PR TITLE
Contact section

### PR DESCRIPTION
**Pull request by kosher9**
- Added contact form and design
- Implemented button interactions
- Added form action link

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -231,6 +231,24 @@ <h4 class="txt-lg">Skills</h4>
     </ul>
 
   </section>
+
+  <section class="contact">
+
+    <h1 class="txt-ctct-me">Contact me</h1>
+    <p class="ctct-me">If you have an application you are interested in developing, a feature that you need built or a
+      project that
+      needs coding. I’d love to help with it </p>
```
Comment (Whoistolu) [inline index.html:240]: - [ ] [OPTIONAL] Kindly ensure that the text is aligned to the center for the `p` and `h1` tags as it has been provided in the design template.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:

Screenshot from 2022-02-15 21-14-37 | Screenshot from 2022-02-15 21-14-12

File: index.html [hunk 2]
```diff
@@ -231,6 +231,24 @@ <h4 class="txt-lg">Skills</h4>
...
+    <form class="container-form" action="https://formspree.io/f/mrgjqpvb" method="post">
+
+      <input class="text-area" type="text" name="name" value="Yeremias" maxlength="30" required placeholder="name">
+      <input class="text-area" type="email" name="email" value="natajayanj14@gmail.com" required placeholder="email">
+      <textarea class="text-area large" placeholder="Write your message" maxlength="500" required></textarea>
+      <input class="btn-container contact-btn" type="submit" name="submit" value="Get In Touch">
+
+    </form>
```
Comment (Whoistolu) [inline index.html:248]: - [ ] [OPTIONAL] Kindly ensure that the input fields and text area are aligned at the center with equal minimal margin to the right and left as provided in the design template. See the image above for more details.
- [ ] [OPTIONAL] Kindly add more right and left padding to the "Get in touch button" as it currently looks smaller than the one provided in the template.

**Review by Whoistolu — APPROVED** [review]
### Hi @kosher9,

Your project is complete!  There is nothing else to say other than... It's time to merge.

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

_Check the inline comments under the review for sections to improve._
- [ ] [OPTIONAL] Please note that it will be more professional to always start your commit messages with an upper case.
### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @Whoistolu if you have any comments.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (kosher9) [reply index.html:248]: Okay. Thank you for your feedback. I will work on it.

Comment (Whoistolu) [reply index.html:248]: You are welcome.

**Pull request merged**